### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HighHowDoIPlay**
Added info about my bot in the "Large Bots" section

**Description:** 

**Justification:**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +135,7 @@ Here are some real-world examples of large bots using Discord4J:
 * [ZeroTwo](https://zerotwo.bot/) - An anime multi-purpose bot used in over 1 million servers.
 * [DisCal](https://www.discalbot.com/) - Implements Google Calendar into Discord as seamlessly and comprehensively as possible; serving over 21k servers.
 * [Shadbot](https://github.com/Shadorc/Shadbot) - A configurable multipurpose bot with music, gambling mini-games, video game stats, and more; serving nearly 12K servers before its shutdown in August 2021.
+* [See-Bot](https://seebot.dev) - A Fortnite focused bot with stat tracking, player lookup, mission alerts, and much more, in over 10k servers
```
Comment (Doc94) [inline README.md:138]:
```suggestion
* [See-Bot](https://seebot.dev) - A Fortnite focused bot with stat tracking, player lookup, mission alerts, and much more; in over 10k servers.
```

**Review by Doc94 — CHANGES_REQUESTED** [review]
Hi,
Can you point this to 3.2.x? (for make the changes in the two current branches)

**Comment by HighHowDoIPlay** [comment]
> Hi, Can you point this to 3.2.x? (for make the changes in the two current branches)

Believe I did, not sure if I did it right.

**Comment by Doc94** [comment]
> > Hi, Can you point this to 3.2.x? (for make the changes in the two current branches)
> 
> Believe I did, not sure if I did it right.

Thanks, yeah the just change cause a masive list of commits added. i get into the branch and make a few changes for clean this.

**Pull request merged**